### PR TITLE
[REVIEW] add ci test for libcudf, libnvstrings headers existence check

### PR DESCRIPTION
**Pull request by karthikeyann**
closes https://github.com/rapidsai/cudf/issues/2167

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: ci/checks/headers_test.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```
Comment (raydouglass) [inline ci/checks/headers_test.sh:1]: This looks good, but CI isn't going to run this check. You could add it to `ci/check/style.sh` if you want to it run with every PR.

Comment (karthikeyann) [reply ci/checks/headers_test.sh:1]: added

**Comment by harrism** [comment]
FWIW, adding a CI check is not what #2167 asked for. It's kind of the opposite. I asked for a way to auto-generate meta.yaml so that it includes all of the headers in the include directory.

**Pull request merged**